x-pack/plugins/enterprise_search/public/applications/shared/flash_messages/flash_messages_logic.ts

```diff
@@ -61,10 +62,10 @@ export const FlashMessagesLogic = kea<MakeLogicType<IFlashMessagesValues, IFlash
       },
     ],
   },
-  events: ({ props, values, actions }) => ({
+  events: ({ values, actions }) => ({
     afterMount: () => {
       // On React Router navigation, clear previous flash messages and load any queued messages
-      const unlisten = props.history.listen(() => {
+      const unlisten = KibanaLogic.values.history.listen(() => {
```

Comment:
I'm not super invested in this change (https://github.com/elastic/kibana/pull/78717/commits/e5f8e15e3de176071daa61e54f8fa663cb9865c5) and would love to get a confidence check from others if they like DRYing out `props: param.history` or nah. Happy to revert if nah!

Reply:
I like this approach, personally.